app/views/how-tos/make-first-prototype/open-prototype-in-editor.html

```diff
@@ -0,0 +1,22 @@
+
+{% set pageTitle = "Open your prototype in your editor" %}
+{% set next = { "title" : "Create pages" , "link" : "create-pages" } %}
+{% set prev = { "title" : "Get started" , "link" : "start" } %}
+
+
+{% extends 'how-tos/make-first-prototype/_BASE.njk' %}
+
+{% block makePrototype %}
+
```

Comment:
Feels odd to go straight from an h1 to an h2. Copied across the content from the GOV.UK tutorial as an intro paragraph.

I also wonder whether an abstract diagram showing the layout of most editors (ie pane on the left with a folder view, and the right hand side having the editor with tabs) would help orientate people?

```suggestion
<p>In your code editor open your prototype folder. You will see the files and folders that you can edit in your prototype.</p>
```